src/main/kotlin/no/nav/syfo/client/pdl/PdlClient.kt

```diff
@@ -52,15 +52,17 @@ class PdlClient(
                 contentType(ContentType.Application.Json)
                 body = request
             }
-            return if (pdlPersonResponse.errors != null && pdlPersonResponse.errors.isNotEmpty()) {
+            if (pdlPersonResponse.errors != null && pdlPersonResponse.errors.isNotEmpty()) {
                 COUNT_CALL_PDL_GT_FAIL.increment()
                 pdlPersonResponse.errors.forEach {
                     log.error("Error while requesting person from PersonDataLosningen: ${it.errorMessage()}")
                 }
-                null
+                throw RuntimeException("No Geografisk Tilknytning was found in response from PDL: Errors found in response")
```

Comment:
Er det ikke bedre å fortsette å returnere null her? Da vil du treffe koden som kaster GeografiskTilknytningMissingException() ovenfor.

Reply:
Tanken er at det skal bli kastet en exception når det er en tilstand som ikke er forventet.
Det er når det er `errors` har innhold, eller når `data er null` uten at `errors` har innhold.

`pdlPersonResponse.data.hentGeografiskTilknytning` derimot kan være null og tilsvarer at det ikke er noe GT for en gitt personIdent, så der returneres det null. 
Men så trengs det en custom exception for dette for å få fanget opp den og returnert 204 i API.

Om du er med på tanken?

Reply:
Prøvde å gjøre noe endringer for å gjøre det lettere å lese koden:
- Slå sammen de to funskjonene for GT i PdlClient og så endret jeg navn på feilklassen til `GeografiskTilknytningNotFoundException`.

Så nå er det `COUNT_CALL_PDL_GT_SUCCESS.increment()` når det er forventede ting som skjer, og `COUNT_CALL_PDL_GT_FAIL.increment()` når det er uforventede ting som skjer.